src/components/scenes/Loans/LoanDetailsScene.tsx

```diff
@@ -174,36 +177,44 @@ export const LoanDetailsScene = (props: Props) => {
         <Space horizontal>
           <Space bottom>
             <SectionHeading>{s.strings.loan_actions_title}</SectionHeading>
+            {isActionProgramRunning ? (
+              <Alert
+                type="warning"
+                title={s.strings.wc_smartcontract_warning_title}
+                message={s.strings.loan_action_program_running}
+                marginRem={[0.5, 0, 0, 0]}
+              />
```

Comment:
Maybe a separate `title` string for this instead of re-using `wc_smartcontract_warning_title`? It could be set to "Waiting on Status" instead of "Warning". What do you think?

Reply:
Good catch, I threw this in just to satisfy the props generically. "Please wait" perhaps?